server/app/views/admin/questions/QuestionEditView.java

```diff
@@ -521,6 +522,12 @@ private DivTag administrativeNameField(String adminName, boolean editExistingQue
         .setId("question-name-input")
         .setFieldName(QUESTION_NAME_FIELD)
         .setLabelText("Administrative identifier. This value can't be changed later")
+        .setToolTipText(
```

Comment:
Do we also want to note that hyphens will be stripped?

Reply:
Yes, good catch.  I'll add that.